### ISSUE TITLE
would be nice to have, get,cookies, now in $smarty

### ISSUE THREAD

**Issue by peon501** [umakantp/jsmart#19]
would be nice to have, get,cookies, now in $smarty

example
{$smarty.get.id}
{$smarty.cookies.user}
{$smarty.now}

**Comment by dekyfin** [comment]
You could pass those variables when calling fetch

**Comment by umakantp** [comment]
Hi @peon501,

There are browser specific features. While jSmart is not just for browser, but common JS environments also. They won't be helpful in those environments anyway. We are trying to keep jSmart as possible as generic for all kinds of environments.

**Comment by hparadiz** [comment]
There should be a function like `jsmart().loadenv()` where we can pass in all the get vars, cookies, etc. That way people could write a plugin for the common nodejs http(s) servers

**Comment by peon501** [comment]
Yes I done prototype for it, but they could implement it easily, but yea :) if No web then no web :D
